waiter/integration/waiter/token_request_test.clj

```diff
@@ -869,4 +882,4 @@
             (delete-service waiter-url service-id)))
 
         (finally
-          (delete-token-and-assert waiter-url token))))))
+          #_(delete-token-and-assert waiter-url token))))))
```

Comment:
Did we mean to remove this before committing?

Reply:
Yes :(